recipe/build.sh

```diff
@@ -0,0 +1,97 @@
+#!/bin/sh
+
+set -ex
+
+if [[ "${target_platform}" == linux-* ]]; then
+  CMAKE_ARGS="
+    ${CMAKE_ARGS}
+    -DQT_FEATURE_egl=ON
+    -DQT_FEATURE_eglfs=ON
+    -DQT_FEATURE_xcb=ON
+    -DQT_FEATURE_xcb_xlib=ON
+    -DQT_FEATURE_xkbcommon=ON
+    -DQT_FEATURE_vulkan=OFF
+    -DQT_FEATURE_wayland=OFF
+  "
+fi
+
+cmake -S"${SRC_DIR}/${PKG_NAME}" -Bbuild -GNinja ${CMAKE_ARGS} \
+  -DCMAKE_PREFIX_PATH=${PREFIX} \
```

Comment:
some comments explaining why you selected some of the most interesting of these flags would be helpful. Especially things that are different from default, or needed to be added to get it to build